lib/extensions/discussion_topic/active_record/base.rb

```diff
@@ -5,11 +5,15 @@ module ClassMethods
     #
     # @option options [Boolean] :display_globally Set to true if the topic need to be displayed in
     # comments center.
-    def acts_as_discussion_topic(display_globally: false)
+    # @option options [Boolean] :touch Set to true if the topic need to be touched upon updating.
+    def acts_as_discussion_topic(display_globally: false, touch: false)
       acts_as :discussion_topic, class_name: Course::Discussion::Topic.name
       # For autoload to work correctly after class changed, we store the model name first and
       # constantize later.
       Course::Discussion::Topic.global_topic_model_names << name if display_globally
+
+      # This can be removed after https://github.com/hzamani/active_record-acts_as/pull/78
+      define_method(:touch_actable) {} unless touch
```

Comment:
This updates the timestamp of the actable class, don't need that for topics.  You can check the details from link above.